templates/learn.html

```diff
@@ -24,7 +24,7 @@
         Bevy Rust API Docs
       </div>
       <div class="card-description">
-        Learn about Bevy's Rust API using the generated Rust documentation.
+        Learn Bevy's Rust API in-depth using the reference documentation. Includes both auto-generated entries and hand-written explanations.
```

Comment:
```suggestion
        Look up information about Bevy's types, methods and traits using the in-depth reference documentation.
```

Reply:
Wouldn't such edit(s) still omit that you can also find a usage information on doc pages? For example, I didn't know you could find [a good manual piece on queries](https://docs.rs/bevy/latest/bevy/ecs/prelude/struct.Query.html) in the docs.

Reply:
```suggestion
Learn how to use Bevy's types, methods and traits using the in-depth reference documentation, complete with inline examples.
```
How about that?

Reply:
Sounds good :)